MapboxNavigation/NavigationViewController.swift

```diff
@@ -589,18 +591,22 @@ extension NavigationViewController: NavigationServiceDelegate {
             currentLeg = progress.currentLeg
             passedApproachingDestinationThreshold = false
             mapViewController?.supressAutomaticAltitudeChanges = false
```

Comment:
Nitpickiest of nits: `supressAutomaticAltitudeChanges` should be spelled `suppressAutomaticAltitudeChanges`, with an extra P.